cmd/otelcol/config.yaml

```diff
@@ -0,0 +1,52 @@
+extensions:
+  health_check:
+  pprof:
+    endpoint: 0.0.0.0:1777
+  zpages:
+    endpoint: 0.0.0.0:55679
+
+receivers:
+
+  opencensus:
+
+  # Collect own metrics
+  prometheus:
+    config:
+      scrape_configs:
+      - job_name: 'otel-collector'
+        scrape_interval: 10s
+        static_configs:
+        - targets: ['0.0.0.0:8888']
+
+  jaeger:
+    protocols:
+      grpc:
+      thrift_binary:
+      thrift_compact:
+      thrift_http:
+
+  zipkin:
+
+processors:
+  batch:
+  queued_retry:
+
+exporters:
+  logging:
+    logLevel: debug
+
+service:
+
+  pipelines:
+
+    traces:
+      receivers: [opencensus, jaeger, zipkin]
```

Comment:
I think we should enable OTLP by default. I think it should be the default or recommended receiver for all data types.